.github/workflows/ci.yml

```
@@ -68,6 +68,8 @@ jobs:
       PHP_EXTENSIONS: none, curl, dom, json, libxml, mbstring, openssl, phar, soap, tokenizer, xml, xmlwriter
       PHP_INI_VALUES: assert.exception=1, zend.assertions=1, error_reporting=-1, log_errors_max_len=0, display_errors=On
 
+    continue-on-error: ${{ matrix.experimental }}
```

Unfortunately, we currently can not use an expression here, for example

```suggestion
    continue-on-error: ${{ matrix.php-version }} == "8.3"
```

For reference, see https://github.com/actions/runner/issues/1492.